views/index.pug

```diff
@@ -0,0 +1,20 @@
+html
+    head
+        body
+            h1 Kim's Convenience
+                
+            h2  Developed By
+                li Ins Choi 
+                li Kevin White
+            h3 Starring
+                li Paul Sun-Hyung Lee
+                li Jean Yoon
+                li Andrea Bang
+                li Simu Liu
+                li Andrew Phung
+                li Nicole Power
+            h4 Seasons
+                li Season 1
```

Comment:
This area wanted you to generate info that would bring you to the second page with hyper links for example:
h2 Seasons
    ul
      for season in showdata.seasons
        li
          a(href=`/season/${season.number}`)= `Season ${season.number}`